src/opera/parser/tosca/csar.py

```diff
@@ -1,31 +1,286 @@
 import shutil
+from abc import ABC, abstractmethod
+from pathlib import Path, PurePath
+from tempfile import mktemp
+from typing import List, IO, Optional
+from zipfile import ZipFile
+
 import yaml
 
-from pathlib import Path
-from tempfile import TemporaryDirectory, mktemp
-from zipfile import ZipFile
+from opera.error import ParseError, OperaError
+from opera.utils import determine_archive_format
```

Comment:
These imports should be split into three sections - import ___, from ___ import ___ , from opera.___ import ___.

So, this should be like this:

```python
import yaml
import shutil

from abc import ABC, abstractmethod
from pathlib import Path, PurePath
from tempfile import mktemp
from typing import List, IO, Optional
from zipfile import ZipFile

from opera.error import ParseError, OperaError
from opera.utils import determine_archive_format
```

Reply:
Why? Is this a coding style specific to this project I'm not aware of? I followed the standard stdlib-lib-project structure.

Reply:
Yeah this is the structure that is mostly used within opera right now. I believe that this comes from following [PEP8](https://www.python.org/dev/peps/pep-0008/) guideline:

![image](https://user-images.githubusercontent.com/41727816/104495088-95884f00-55d7-11eb-9776-52775696abaa.png)

Reply:
Exactly, the guideline is what I used. If you want, I can change it to your style, but it isn't PEP8.

Reply:
Ok, we can then keep that. All in all, I think that my suggestion for the imports above just looks nicer, but since we are still trying to stick to PEP8 this is ok.